documentation/src/main/asciidoc/stories/assembly_hot_rod_marshalling.adoc

```diff
@@ -0,0 +1,19 @@
+[id='hot_rod_marshalling']
+:context: encoding
+= Hot Rod Java client marshalling
+
+Serialization::
+is the process of translating a data structure or object state into a format that can be stored or transmitted and reconstructed later.
+When the resulting series of bits is reread according to the serialization format, it can be used to create a semantically identical clone of the original object.
+
+Marshalling::
+is a serialization process that transforms the memory representation of an object to a data format suitable for storage or transmission,and it is typically used when data must be moved between different parts of a computer program or from one program to another. It simplifies complex communication, using composite objects in order to communicate instead of primitives.
```

Comment:
```suggestion
is a serialization process that transforms the memory representation of an object to a data format suitable for storage or transmission, and it is typically used when data must be moved between different parts of a computer program or from one program to another. It simplifies complex communication, using composite objects in order to communicate instead of primitives.
```